code/datums/supplypacks/hospitality_vr.dm

```diff
@@ -0,0 +1,143 @@
+/datum/supply_packs/randomised/hospitality/cuisine_vr
+	num_contained = 5
+	contains = list(
+			/obj/item/weapon/reagent_containers/food/snacks/blackpudding,
+			/obj/item/weapon/reagent_containers/food/snacks/bangersandmash,
+			/obj/item/weapon/reagent_containers/food/snacks/cubancarp,
+			/obj/item/weapon/reagent_containers/food/snacks/cubannachos,
+			/obj/item/weapon/reagent_containers/food/snacks/dionaroast,
+			/obj/item/weapon/reagent_containers/food/snacks/donerkebab,
+			/obj/item/weapon/reagent_containers/food/snacks/goulash,
+			/obj/item/weapon/reagent_containers/food/snacks/kitsuneudon,
+			/obj/item/weapon/reagent_containers/food/snacks/makaroni,
+			/obj/item/weapon/reagent_containers/food/snacks/mammi,
+			/obj/item/weapon/reagent_containers/food/snacks/milosoup,
+			/obj/item/weapon/reagent_containers/food/snacks/pillbug,
+			/obj/item/weapon/reagent_containers/food/snacks/sashimi,
+			/obj/item/weapon/reagent_containers/food/snacks/sliceable/sushi,
+			/obj/item/weapon/reagent_containers/food/snacks/benedict,
+			/obj/item/weapon/reagent_containers/food/snacks/enchiladas,
+			/obj/item/weapon/reagent_containers/food/snacks/cheeseburrito,
+			/obj/item/weapon/reagent_containers/food/snacks/cheesenachos,
+			/obj/item/weapon/reagent_containers/food/snacks/eggplantparm,
+			/obj/item/weapon/reagent_containers/food/snacks/fuegoburrito,
+			/obj/item/weapon/reagent_containers/food/snacks/generalschicken,
+			/obj/item/weapon/reagent_containers/food/snacks/hotandsoursoup,
+			/obj/item/weapon/reagent_containers/food/snacks/hotchili,
+			/obj/item/weapon/reagent_containers/food/snacks/lasagna,
+			/obj/item/weapon/reagent_containers/food/snacks/meatburrito,
+			/obj/item/weapon/reagent_containers/food/snacks/wingfangchu
+			)
+	name = "Cuisine crate"
+	cost = 25
+	containertype = /obj/structure/closet/crate/freezer
+	containername = "Cuisine crate"
+
+/datum/supply_packs/randomised/hospitality/burgers_vr
+	num_contained = 5
```

Comment:
This line is redundant. Only 5 items can appear in the crate at all.

Reply:
I think that would still randomize the contents to not necessarily have just one of each.

Reply:
Good point. Very well then.